Jenkinsfile

```diff
@@ -1,5 +1,3 @@
-@Library('cicd-lib@0.11') _
```

Comment:
Why do you remove this? It is necessary for 'Publish ingeniamotion' step

Reply:
Done